code/__HELPERS/sanitize_values.dm

```diff
@@ -7,6 +7,14 @@
 			return number
 	return default
 
+/// Return `float` if it is in the range `min to max`, otherwise `default`
+/proc/sanitize_float(number, min=0, max=1, accuracy=1, default=0)
```

Comment:
```suggestion
/proc/sanitize_float(number, min=0, max=1, accuracy=0.5, default=0)
```
I kind of feel like it should default to an accuracy of 0.5 if it's intended for floats

Reply:
Actually 0.1 might be better, I'm not sure.